src/main/java/com/google/sps/user/User.java

```diff
@@ -0,0 +1,81 @@
+package com.google.sps.user;
+
+import org.jetbrains.annotations.Nullable;
+
+/** Stores the data related to one user. */
+public class User {
+    private final String id, email;
```

Comment:
Define each variable in separated line. It is more readable

Reply:
Done.